src/dialogs/config-entry-system-options/dialog-config-entry-system-options.ts

```diff
@@ -60,7 +60,13 @@ class DialogConfigEntrySystemOptions extends LitElement {
         @opened-changed="${this._openedChanged}"
       >
         <h2>
-          ${this.hass.localize("ui.dialogs.config_entry_system_options.title")}
+          ${this.hass.localize(
+            "ui.dialogs.config_entry_system_options.title",
+            "integration",
+            this.hass.localize(
+              `component.${this._params.entry.domain}.config.title`
+            )
```

Comment:
It could be that there is not translation:
```
this.hass.localize(`component.${this._params.entry.domain}.config.title`) || ${this._params.entry.domain}
```

Reply:
Good point